src/app/course/course.module.ts

```diff
@@ -201,15 +201,16 @@ import {ConceptListComponent} from './concept-list/concept-list.component'
         TuiPointerHintModule,
         TuiProgressModule,
         TuiRingChartModule,
+        TuiScrollbarModule,
         TuiSelectModule,
         TuiSidebarModule,
         TuiStepperModule,
         TuiSvgModule,
         TuiTableModule,
         TuiTabsModule,
         TuiTagModule,
-        TuiTextAreaModule,
```

Comment:
try to move random stuff around

Reply:
Honestly the linting just did that as I was fixing the placement of the scrollbar import but I can move it back